testing/connectsdk.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+set -e
+# TODO: No need to wait until we're doing actual integration testing
+CI=true npm --prefix ../sdk/js-connect run test
+
+# num=${NUM_GUARDIANS:-1} # default value for NUM_GUARDIANS = 1
+# for ((i=0; i<num; i++)); do
+#     while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' guardian-$i.guardian:6060/readyz)" != "200" ]]; do sleep 5; echo "waiting for guardian $i"; done
+# done
+# 
+# while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' spy:6060/metrics)" != "200" ]]; do sleep 5; done
+# while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' ibc-relayer:7597/debug/pprof/)" != "200" ]]; do sleep 5; done
+# while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' relayer-engine:3000/metrics)" != "200" ]]; do sleep 5; done
+#CI=true npm --prefix ../sdk/js-connect run test-ci
```

Comment:
Unless there is a really good reason, don't commit commented out dead code. If there is, commit it and then remove it in the next commit. That way it is in version control and can be dug up trivially using some git foo that first requires sacrificing the a goat to the git gods.
```suggestion
```